circle.yml

```diff
@@ -137,6 +137,10 @@ jobs:
         command: |
           set +e
           ulimit -s 10000
+          # RUNTIME_TEST_HACK: This ugly hack works around all the special cases
```

Comment:
I think I can live with this hack, at least until we can make `runtime` tests pass :)

However, I can propose an alternative hack: make `parseCallstack` exported and move the test to gopherjs/tests package. Since it will be hidden in the overlay sources, I doubt that anyone will ever discover it or try to use it for anything and it would be a bit less of a special case. A stern warning not to use it may also help the situation.

There is probably also a way to test it by getting the current call stack and examining `runtime.CallersFrames()`, but in that case it's harder to know what the expected correct values would be, so I'm not really proposing that.

Reply:
Nice suggestion.  It worked pretty smoothly.